README.md

```diff
@@ -67,6 +69,8 @@ selinux_policy: targeted
 selinux_state: enforcing
 ```
 
```

Comment:
It should be said that if `selinux_state` is not given, it is not changed (the previous value is preserved).

Reply:
amended